pkgs/by-name/zo/zoom-us/package.nix

```
@@ -220,6 +228,7 @@ stdenv.mkDerivation {
   dontPatchELF = true;
 
   passthru.updateScript = ./update.sh;
+  passthru.tests.startwindow = callPackage ./test.nix { };
```

Future PR: this should do something like this, so that it tests the derivation that it's attached to in the presence of overrides.

```suggestion
  passthru.tests.startwindow = callPackage ./test.nix {
    zoom-us = finalAttrs.finalPackage;
  };
```